src/main/scala/com/exasol/spark/DefaultSource.scala

```diff
@@ -85,24 +85,26 @@ class DefaultSource
     parameters: Map[String, String],
     data: DataFrame
   ): BaseRelation = {
-    val tableName = getKValue("table", parameters)
+    val tableName = getKeyValue("table", parameters)
     val manager = createManager(parameters, sqlContext)
-
+    if (manager.config.drop_table) {
+      manager.dropTable(tableName)
```

Comment:
or `replaceExisting`

Reply:
Yes, it is related but your suggestion already covered. 

In Spark, users can save a dataframe into an Exasol table. Usually, the api is as below:
```
dataframe
      .write
      .mode("override")
      .option("host", "ip-address")
      .option("drop_table", "true")
      .format("exasol")
      .save()
```

The above change is about the user options (provided by `.option`), the mode value can be "override", "append", "error". 
With mode override we do truncate the table if it is available. This change adds user option to drop it before saving, because the assumption is that user knows that dataframe changed and is different than the exising table.

Reply:
In summary, user wants to really drop previously existing table before saving.